.github/workflows/update-stored-dependencies-version.yaml

```diff
@@ -18,7 +18,6 @@ jobs:
   get-latest-sha:
     name: Get projects latest SHA
     runs-on: ubuntu-latest
-    shell: bash
     permissions:
       actions_variables: write
```

Comment:
Is this a new (undocumented) job permission? https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
The issue now is....   

```
"message": "Resource not accessible by integration",
```

Reply:
Ok, it's not a permission issue, but a var interpolation one...